puppet/modules/prometheus/manifests/node_exporter.pp

```diff
@@ -94,11 +68,41 @@
       }
     }
 
-    kubernetes::apply{'node-exporter':
-      manifests => [
-        template('prometheus/prometheus-ns.yaml.erb'),
-        template('prometheus/node-exporter-ds.yaml.erb'),
-      ],
+    $external_scrape_targets_only = $::prometheus::external_scrape_targets_only
+    if ! $external_scrape_targets_only {
+      kubernetes::apply{'node-exporter':
+        manifests => [
+          template('prometheus/prometheus-ns.yaml.erb'),
+          template('prometheus/node-exporter-ds.yaml.erb'),
```

Comment:
we include node_exporter when `! $external_scrape_targets_only` is true so I assume this apply should be before the `! $external_scrape_targets_only` check too?

Reply:
No we always include node_exporter now, but only if external_scrape_targets_only is not true we want to have the daemonset node exporters